src/detail-page/components/charts/DevOpsChart.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { VFC } from 'react';
+import { Pie } from 'react-chartjs-2';
+import styled from 'styled-components';
+import { devOps } from '../../../data/detail-page/charts';
+import { chartColors } from '../../../data/detail-page/charts/chartColors';
+
+const { data: sourceData, labelText, source, title: chartTitle } = devOps;
+
+const ChartData = {
+  labels: sourceData?.map(({ label }) => label),
+  datasets: [
+    {
+      label: labelText,
+      data: sourceData?.map(({ value }) => value),
+      backgroundColor: sourceData?.map(({ color }) => chartColors[color].light),
+      borderColor: sourceData?.map(({ color }) => chartColors[color].dark),
+      borderWidth: 1,
+    },
+  ],
+};
+
+const ChartOptions = {
+  responsive: true,
+  title: {
+    display: true,
+    text: chartTitle,
+  },
+  legend: {
+    display: true,
+  },
+};
+
+const StyledWrapper = styled('div')`
+  max-width: 400px;
+`;
+
+const StyledSource = styled('p')`
+  font-size: 0.8rem;
+  margin-top: ${({ theme }) => `${theme.spacing(3)}px`};
+  text-align: right;
+`;
+
+// Source Paragraph can be shared between all charts?
+export const DevOpsChart: VFC = () => (
```

Comment:
I'm always impressed how you manage to find interesting bits and pieces in the docs 👏 In this case, `VFC` seems like a more precise definition for our component, but from what I found it looks like [it's a temporary solution](https://basarat.gitbook.io/typescript/tsx/react#void-function-components), so we should probably go with the usual `FC` instead.

Reply:
In the next major release there will be probably a change with `FC` component too  :smile: but because we didn't use `VFC` anywhere else you're right that we should get back to `FC`.